devtools/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionSha256Sum=f30b29580fe11719087d698da23f3b0f0d04031d8995f7dd8275a31f7674dc01
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-all.zip
+networkTimeout=10000
```

Comment:
`networkTimeout` is added when bumping the wrapper version using `./gradlew wrapper ...` - it was always kinda recommended to set it.